src/components/Login/Login.jsx

```diff
@@ -38,15 +59,18 @@ function Login() {
 								Username
 								<input
 									className={styles.btnUser}
-									id="username"
-									placeholder="Username"
-									type="text"
+									name="email"
 									{...register('email', {
 										required: {
 											message: 'El email es obligatorio',
 											value: true,
 										},
 									})}
+									// eslint-disable-next-line react/jsx-no-bind
+									onChange={inputChange}
```

Comment:
Seems that I put wrong the rule in eslint, please dissable react/jsx-no-bind from there, and remove this comment